dressForTheWeather/Models/TemperatureRanges.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  TemperatureRanges.swift
+//  dressForTheWeather
+//
+//  Created by Leeann Drees on 6/17/19.
+//  Copyright © 2019 DetroitLabs. All rights reserved.
+//
+
+import Foundation
+
+enum TemperatureRanges: String, CaseIterable {
```

Comment:
CaseIterable isn't being used, so it can be removed

Reply:
👍 Done in 8c056ebcfcb25022802ea92acad2a84eb81be949